react/features/toolbox/components/OverflowMenuItem.web.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+/**
+ * A React {@code Component} for displaying a link to interact with other
+ * features of the application.
+ *
+ * @extends Component
+ */
+class OverflowMenuItem extends Component {
+    static propTypes = {
+        /**
+         * The icon class to use for displaying an icon before the link text.
+         */
+        icon: PropTypes.string,
+
+        /**
+         * The callback to invoke when {@code OverflowMenuItem} is clicked.
+         */
+        onClick: PropTypes.func,
+
+        /**
+         * The text to display in the {@code OverflowMenuItem}.
+         */
+        text: PropTypes.string
+    };
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
```

Comment:
Probably not. Inheritdoc solves a lot of jsdoc related linting errors. The @returns in render() has been how the render() doc has been written. Probably we all copy paste from an existing one because the comment is almost always the same.

Reply:
Probably true :)